libs/ledger-live-common/src/cal/partners.ts

```diff
@@ -49,11 +50,11 @@ export type ProvidersDataResponse = {
 // Exported for test purpose only
 export function transformData(
   providersData: ProvidersDataResponse,
-  env: "prod" | "test",
 ): Record<string, ExchangeProvider> {
+  const env = getEnv("MOCK_EXCHANGE_TEST_CONFIG") ? "test" : "prod";
```

Comment:
I disagree the usage of the `getEnv` function in this file. The `env` parameter` was meant to pass what value is needed, based on how we call those functions.

You should add this `getEnv` call in the `providers/swap::getSwapProvider` and `providers/index::getProviderConfig` functions. Frankly I don't recall why I didn't do it.

Reply:
I do not think bring any value pass a env variable across multiple functions...

Reply:
Changed.